crates/server/src/graphql/modules/post/mutation/post_create.rs

```diff
@@ -0,0 +1,51 @@
+use async_graphql::{Context, Result};
+use gabble::post::service::CreatePostDto;
+use pxid::Pxid;
+use serde::{Deserialize, Serialize};
+
+use crate::{
+    context::SharedContext,
+    graphql::modules::post::types::{Post, PostError, PostErrorCode},
+};
+
+#[derive(Debug, Default)]
+pub struct PostCreateInput {
+    pub author_id: String,
+    pub parent_id: Option<Pxid>,
```

Comment:
Here you want to use `ID` from `async_graphql` because you are on GraphQL layer.

```suggestion
    pub author_id: ID,
    pub parent_id: Option<ID>,
```